packages/site/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@comet/brevo-site",
+    "version": "1.0.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/vivid-planet/comet",
+        "directory": "packages/site"
+    },
+    "license": "BSD-2-Clause",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "files": [
+        "lib/*"
+    ],
+    "scripts": {
+        "build": "$npm_execpath run clean && tsc -p tsconfig.build.json",
+        "clean": "rimraf lib",
+        "dev": "tsc --watch --preserveWatchOutput -p tsconfig.build.json",
+        "generate-schema": "ts-node generate-schema.ts",
+        "generate-schema:watch": "chokidar \"src/\" -c \"$npm_execpath generate-schema\"",
+        "lint": "run-p lint:eslint lint:tsc",
+        "lint:eslint": "eslint --max-warnings 0 src/ package.json",
+        "lint:prettier": "npx prettier --check './**/*.{js,json,md,yml,yaml}'",
+        "lint:tsc": "tsc --noEmit"
+    },
+    "dependencies": {
+        "rimraf": "^3.0.0"
+    },
+    "devDependencies": {
+        "@cspell/eslint-plugin": "^8.16.0",
+        "eslint": "^8.0.0"
+    },
+    "peerDependencies": {
+        "react": "^18.0.0",
+        "react-dom": "^18.0.0",
+        "react-intl": "^5.0.0 || ^6.0.0",
+        "styled-components": "^6.0.0"
```

Comment:
Peer dependenies need to be installed as dev dependency as well. Also, I'd remove react-intl and styled-components for now and only add the once they're needed.